http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/Config.scala

```diff
@@ -606,7 +606,8 @@ object Config {
     CustomAppConfig(
       config.as[ChartName]("chartName"),
       config.as[ChartVersion]("chartVersion"),
-      config.as[String]("releaseNameSuffix")
+      config.as[String]("releaseNameSuffix"),
+      config.as[String]("namespaceNameSuffix")
```

Comment:
you might have some merge conflicts with https://github.com/DataBiosphere/leonardo/pull/2354

Reply:
Definitely will have merge conflicts! I figured I would do the rebase/conflict resolution of this branch after mine merges (@cjllanwarne is out until Wednesday).